code/modules/events/disease_outbreak.dm

```diff
@@ -27,11 +27,17 @@
 			continue
 		if(T.z != ZLEVEL_STATION)
 			continue
-		var/foundAlready = 0	// don't infect someone that already has the virus
-		for(var/datum/disease/D in H.viruses)
-			foundAlready = 1
+		if(!H.client)

```

Comment:
I believe lack of this check was intentional, maybe not interesting but intended.